config/locales/en/claims/schools/mentors.yml

```diff
@@ -2,6 +2,12 @@ en:
   claims:
     schools:
       mentors:
+        add_mentor:
+          edit:
+            caption: Add mentor
+            cancel: Cancel
+          update:
+            success_heading: Mentor added
```

Comment:
Personal preference (feel free to ignore): In our other implementations we've separated this out into separate locale files.

Reply:
@JamieCleare2525 definitely happy to separate this out, but comparing to the analogous `placements/schools/mentors.yml` I'm not sure exactly what you mean 😄